test/modules/ssh-accounts-integration.ts

```diff
@@ -193,6 +193,8 @@ describe('EC2 Integration Testing', () => {
 
   it('installs the ssh_accounts and aws_ec2_metadata modules', install(['ssh_accounts', 'aws_ec2_metadata']));
 
+  it('starts a transaction', begin());
```

Comment:
This shouldn't matter because the `ssh_credentials` table, like the `aws_credentials` table, doesn't actually sync with anything in any cloud.